### DISCUSSION TITLE
请问哪里可以找到带符号表的so，想排查native层crash

### DISCUSSION THREAD

**Discussion by syzhugh** [Tencent/MMKV#977] [Q&A] [answered]
*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
pid: 0, tid: 0 >>> *** <<<

backtrace:
  #00  pc 0x0000000000024dbc  /data/app/~~CZQ0Chn7mfzDAWopbl3lzw==/***9HfeuCYY8jU7S5nFscp8Cg==/split_config.arm64_v8a.apk!libmmkv.so
  #01  pc 0x000000000001906c  /data/app/~~CZQ0Chn7mfzDAWopbl3lzw==/***-9HfeuCYY8jU7S5nFscp8Cg==/split_config.arm64_v8a.apk!libmmkv.so
  #02  pc 0x0000000000011cec  /data/app/~~CZQ0Chn7mfzDAWopbl3lzw==/***-9HfeuCYY8jU7S5nFscp8Cg==/split_config.arm64_v8a.apk!libmmkv.so

**Comment by lingol** [comment, marked as answer]
I just don't understand why Android developers keep asking **a very basic question**. 
It's on Maven Central. Just Google about 'maven central mmkv':
https://search.maven.org/artifact/com.tencent/mmkv-static/1.2.12/aar
https://repo1.maven.org/maven2/com/tencent/mmkv-static/1.2.12/

**Reply by syzhugh** [reply]
请问1.2.2版本的还有办法找到吗，感谢

**Reply by lingol** [reply]
No, you can't. https://github.com/Tencent/MMKV/releases/tag/v1.2.8
You should upgrade your MMKV instead.
